jovian/jovian_nb_ext/main.js

```diff
@@ -516,7 +516,105 @@ define([
       return form;
     };
 
-    const saveParams = function() {
+    const settingsUI = function() {
+      /**
+       * Body of the Form
+       *
+       * Layout:
+       *  - div :
+       *    - label : text: Set Default Commit Parameters
+       *    - input : type: button | id: default_param_button | class: btn btn-primary | text: Set Default | title: Open Parameter Window to set Default Parameters
+       * 
+       *    - label : text: Clear API Key
+       *    - input : type: button | id: clear_api_button | class: btn btn-primary | text: Clear Key | title: Clear the Jovian API key
+       *
+       *    - label : text: Change API Key
+       *    - input : type: button | id: change_api_button | class: btn btn-primary | text: Change Key | title: Change the Jovian API key
+       *
+       *    - label : text: Disable Jovian Extension
+       *    - input : type: button | id: disable_button | class: btn btn-primary | text: Disable | title: Disable the Jovian Extension
+       *
+       */
+      const div = $("<div/>").attr("id", "input_div");
+
+      const label1 = $("<label/>").text("Set Default Commit Parameters");
+
+      const option1 = $("<button/>")
+        .attr("id", "default_param_button")
+        .addClass("btn btn-primary")
+        .text("Set Default")
+        .attr("title", "Open Parameter Window to set Default Parameters");
+
+      const label2 = $("<label/>").text("Clear API Key");
+
+      const option2 = $("<button/>")
+        .attr("id", "clear_api_button")
+        .addClass("btn btn-primary")
+        .text("Clear")
+        .attr("title", "Clear the Jovian API key");
+
+      const label3 = $("<label/>").text("Change API Key");
+
+      const option3 = $("<button/>")
+        .attr("id", "change_api_button")
+        .addClass("btn btn-primary")
+        .text("Change")
+        .attr("title", "Change the Jovian API key");
+
+      const label4 = $("<label/>").text("Disable Jovian Extension");
+
+      const option4 = $("<button/>")
+        .attr("id", "disable_button")
+        .addClass("btn btn-primary")
+        .text("Disable")
+        .attr("title", "Disable the Jovian Extension");
+
+      div
+        .append(label1)
+        .append(option1)
+        .append("</br>")
+        .append(label2)
+        .append(option2)
+        .append("</br>")
+        .append(label3)
+        .append(option3)
+        .append("</br>")
+        .append(label4)
+        .append(option4);
```

Comment:
Combine each label and option to a single constant.

Also provide appropriate names instead of generic names like label1, label2

Reply:
renamed and combined